test_historgram.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import cv2
+
+
+def plot_histogram(image):
+    im = cv2.imread(image)
+    # print("im type: ", type(im))
+    vals = im.mean(axis=2).flatten()
+    counts, bins = np.histogram(vals, range(257))
+    plt.bar(bins[:-1] - 0.5, counts, width=1, edgecolor='none')
+    plt.xlim([-0.5, 255.5])
+    plt.savefig('test.JPEG')
+    #plt.show()
+
+
+
+def plot_histogram_rgb(image):
+    im = cv2.imread(image)
+    color = ('b', 'g', 'r')
+    for i, col in enumerate(color):
+        histr = cv2.calcHist([im], [i], None, [256], [0, 256])
+        plt.plot(histr, color=col)
+        plt.xlim([0, 256])
+    plt.show()
+
+
+img_one_ch = 'app/static/imagenet_subset/n01443537_goldfish.JPEG'
+# print(img_one_ch)
```

Comment:
I think these are leftover comments that can be removed. @davide-scintu Can you fix this?